.env

```diff
@@ -0,0 +1 @@
+REACT_APP_RELAY_API_URL=https://safe-relay.dev.gnosisdev.com/api/v1
```

Comment:
Instead of adding a new .end file, I suggest adding this URL with the other ones.
Here you can see how it's done with the  Transaction Service `getTxServiceHost()`
https://github.com/gnosis/safe-react/blob/master/src/config/index.js#L26
You should add it to the files `development.js`, `production.js` and `testing.js`) here: https://github.com/gnosis/safe-react/tree/master/src/config